orttraining/orttraining/test/python/checkpoint/orttraining_test_save_checkpoint.py

```diff
@@ -0,0 +1,222 @@
+#!/usr/bin/env python3
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import os
+import pickle
+import argparse
+from itertools import islice
+
+import torch
+import torch.distributed as dist
+
+from onnxruntime import set_seed
+from onnxruntime.training import amp, optim, orttrainer
+from _test_helpers import distributed_setup, _load_pytorch_transformer_model, train, save, chunkify
+
+def single_node_full_precision(device = 'cuda', checkpoint_dir = 'checkpoint_dir/single_node/full_precision/'):
+    learning_rate = 0.1
+    seed = 1
+
+    torch.manual_seed(seed)
+    set_seed(seed)
+
+    # PyTorch transformer model as example
+    opts = {'device' : {'id' : device},
+            'debug' : {'deterministic_compute': True}}
+    opts = orttrainer.ORTTrainerOptions(opts)
+    optim_config = optim.LambConfig(lr=learning_rate)
+    model, model_desc, loss_fn, batcher_fn, train_data, _, _ = _load_pytorch_transformer_model(device)
+    trainer = orttrainer.ORTTrainer(model, model_desc, optim_config, loss_fn=loss_fn, options=opts)
+
+    # run train steps
+    train(trainer, train_data, batcher_fn)
```

Comment:
Just like you did for the load part, put all this in a helper and save lots of typing :)